src/ecco/output.py

```diff
@@ -237,6 +240,9 @@ def saliency(self, attr_method: Optional[str] = 'grad_x_input', style="minimal",
 
         importance_id = position - self.n_input_tokens
```

Comment:
@jalammar this `importance` variable is used to calculate the `imp` of each token, but I don't think it is used anywhere, is it?

Reply:
I just removed it and seems that the JS code is not looking at it. I will remove it in another PR if that is okay :) 
We are adding LIME attributions

Reply:
In this method it is always set to zero, but should be dynamic, depending on the position chosen in the UI (when we hover with the mouse), correct?

Reply:
Yeah, spot on. I think that is an artifact of an old implementation. Currently the values for all the tokens are passed to the JS and it's shifted from there (no logic goes back to python land after the visualization is created). I think this may need to be refactored to indicate this more clearly.